WasatchNET/Spectrometer.cs

```diff
@@ -2744,6 +2751,8 @@ public virtual double[] getSpectrum(bool forceNew=false)
                 }
 
                 correctBadPixels(ref sum);
+                if (useRamanIntensityCorrection)
+                    sum = correctRamanIntensity(sum);

```

Comment:
So...we're rescaling the spectrum in Y...before subtracting dark.  I guess this is assuming that the dark was already similarly SRM-corrected, even though the dark has no Raman signal?

Reply:
I think this is a little tricky. In enlighten I believe you must take a dark before correction is enabled, but I don't know what happens if you turn on correction and then retake a dark.

I agree with the sentiment that the dark shouldn't be corrected since it isn't raman.

Ideally the program would just "know" if the dark were corrected or not, but since the driver is lightweight and I don't want to add that structure, whichever way we do it a user could potentially run themselves into a corner.

I'll switch the order and add a note above the bool switch

Reply:
I think I did it this way so that people could "set and forget" but I agree dark first is more valid